build.sbt

```diff
@@ -616,18 +667,17 @@ lazy val test = project
     fork in IntegrationTest := true,
     javaOptions in IntegrationTest += "-Xmx1G",
     testFrameworks += new TestFramework("scala.tools.partest.sbt.Framework"),
-    testOptions in IntegrationTest += Tests.Setup( () => root.base.getAbsolutePath + "/pull-binary-libs.sh" ! ),
```

Comment:
so when I run
```
sbt
> partest test/files/pos/cycle-jsoup.scala
```
on a fresh checkout (or after `git clean -dfx`), I get:
```
...
[info] Packaging /Users/luc/scala/scala4/build/pack/lib/scala-partest-javaagent.jar ...
[info] Done packaging.
...
Resolving [346d3dff4088839d6b4d163efa2892124039d216/test/files/lib/jsoup-1.3.1.jar]
...
Partest version:
Compiler under test: $baseDir/compiler
Scala version is:    Scala compiler version 2.12.0-20160606-115156-79e7334 -- Copyright 2002-2016, LAMP/EPFL
Scalac options are:
Compilation Path:    /Users/luc/scala/scala4/target/test/it-classes:$baseDir/test:$baseDir/compiler:$baseDir/library:$baseDir/reflect:$baseDir/interactive:$baseDir/repl-jline-embedded:$baseDir/repl-jline:$baseDir/repl:$baseDir/scalap:$baseDir/partest-extras:/Users/luc/scala/scala4/build/pack/lib/scala-partest-javaagent.jar:$baseDir/scaladoc:/Users/luc/scala/scala4/test/lib:/Users/luc/.ivy2/cache/org.apache.ant/ant/jars/ant-1.9.4.jar:/Users/luc/.ivy2/cache/org.apache.ant/ant-launcher/jars/ant-launcher-1.9.4.jar:/Users/luc/.ivy2/cache/org.scala-lang.modules/scala-asm/bundles/scala-asm-5.0.4-scala-3.jar:/Users/luc/.ivy2/cache/org.scala-lang.modules/scala-xml_2.12.0-M4/bundles/scala-xml_2.12.0-M4-1.0.5.jar:/Users/luc/.ivy2/cache/org.scala-lang.modules/scala-parser-combinators_2.12.0-M4/bundles/scala-parser-combinators_2.12.0-M4-1.0.4.jar:/Users/luc/.ivy2/cache/jline/jline/jars/jline-2.14.1.jar:/Users/luc/.ivy2/cache/org.scala-lang.modules/scala-partest_2.12.0-M4/jars/scala-partest_2.12.0-M4-1.0.14.jar:/Users/luc/.ivy2/cache/com.googlecode.java-diff-utils/diffutils/jars/diffutils-1.3.0.jar:/Users/luc/.ivy2/cache/org.scala-sbt/test-interface/jars/test-interface-1.0.jar:/Users/luc/.ivy2/cache/org.scalacheck/scalacheck_2.12.0-M4/jars/scalacheck_2.12.0-M4-1.11.6.jar:/Users/luc/.sbt/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.11/test-agent-0.13.11.jar:/Users/luc/.sbt/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.11/test-interface-1.0.jar
Java binaries in:    /Library/Java/JavaVirtualMachines/jdk1.8.0_77.jdk/Contents/Home/jre/bin
Java runtime is:     Java HotSpot(TM) 64-Bit Server VM (build 25.77-b03, mixed mode)
Java options are:    -Xmx1024M -Xms64M -XX:MaxPermSize=128M
baseDir:             /Users/luc/scala/scala4/build/quick/classes
sourceDir:           /Users/luc/scala/scala4/test/files

Selected 1 tests drawn from specified tests

# starting 1 test in pos
!! 1 - pos/cycle-jsoup.scala                     [compilation failed]
```
with
```
cycle-jsoup.scala:3: error: object jsoup is not a member of package org
    org.jsoup.Jsoup.parse(null: java.net.URL, 3000)
        ^
one error found
```

If I then run the same command `partest test/files/pos/cycle-jsoup.scala` again in the same sbt shell process (no restart), the test passes:

```
> partest test/files/pos/cycle-jsoup.scala --show-diff
[info] Packaging /Users/luc/scala/scala4/build/pack/lib/scala-partest-javaagent.jar ...
[info] Done packaging.
Partest version:
Compiler under test: $baseDir/compiler
Scala version is:    Scala compiler version 2.12.0-20160606-115156-79e7334 -- Copyright 2002-2016, LAMP/EPFL
Scalac options are:
Compilation Path:    /Users/luc/scala/scala4/target/test/it-classes:$baseDir/test:$baseDir/compiler:$baseDir/library:$baseDir/reflect:$baseDir/interactive:$baseDir/repl-jline-embedded:$baseDir/repl-jline:$baseDir/repl:$baseDir/scalap:$baseDir/partest-extras:/Users/luc/scala/scala4/build/pack/lib/scala-partest-javaagent.jar:$baseDir/scaladoc:$sourceDir/lib/annotations.jar:$sourceDir/lib/enums.jar:$sourceDir/lib/genericNest.jar:$sourceDir/lib/jsoup-1.3.1.jar:$sourceDir/lib/macro210.jar:$sourceDir/lib/methvsfield.jar:$sourceDir/lib/nest.jar:/Users/luc/scala/scala4/test/lib:/Users/luc/.ivy2/cache/org.apache.ant/ant/jars/ant-1.9.4.jar:/Users/luc/.ivy2/cache/org.apache.ant/ant-launcher/jars/ant-launcher-1.9.4.jar:/Users/luc/.ivy2/cache/org.scala-lang.modules/scala-asm/bundles/scala-asm-5.0.4-scala-3.jar:/Users/luc/.ivy2/cache/org.scala-lang.modules/scala-xml_2.12.0-M4/bundles/scala-xml_2.12.0-M4-1.0.5.jar:/Users/luc/.ivy2/cache/org.scala-lang.modules/scala-parser-combinators_2.12.0-M4/bundles/scala-parser-combinators_2.12.0-M4-1.0.4.jar:/Users/luc/.ivy2/cache/jline/jline/jars/jline-2.14.1.jar:/Users/luc/.ivy2/cache/org.scala-lang.modules/scala-partest_2.12.0-M4/jars/scala-partest_2.12.0-M4-1.0.14.jar:/Users/luc/.ivy2/cache/com.googlecode.java-diff-utils/diffutils/jars/diffutils-1.3.0.jar:/Users/luc/.ivy2/cache/org.scala-sbt/test-interface/jars/test-interface-1.0.jar:/Users/luc/.ivy2/cache/org.scalacheck/scalacheck_2.12.0-M4/jars/scalacheck_2.12.0-M4-1.11.6.jar:/Users/luc/.sbt/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.11/test-agent-0.13.11.jar:/Users/luc/.sbt/boot/scala-2.10.6/org.scala-sbt/sbt/0.13.11/test-interface-1.0.jar
Java binaries in:    /Library/Java/JavaVirtualMachines/jdk1.8.0_77.jdk/Contents/Home/jre/bin
Java runtime is:     Java HotSpot(TM) 64-Bit Server VM (build 25.77-b03, mixed mode)
Java options are:    -Xmx1024M -Xms64M -XX:MaxPermSize=128M
baseDir:             /Users/luc/scala/scala4/build/quick/classes
sourceDir:           /Users/luc/scala/scala4/test/files

Selected 1 tests drawn from specified tests

# starting 1 test in pos
ok 1 - pos/cycle-jsoup.scala
```

note that `jsoup-1.3.1.jar` is only on the "Compilation Path" in the second invocation.